packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js

```diff
@@ -1,219 +1,293 @@
+          chosenLanguage: mockData.locales[0].name,
+          chosenStack: mockData.stacks[0].name,
+          chosenContentTypes: [mockData.contentTypes[0].uid],
+          branch: mockData.branch.uid,
+        });
+      })
+      .nock(cma, (api) => {
+        api.get(`/v3/organizations?limit=100`).reply(200, { organizations: mockData.organizations });
+      })
+      .nock(cma, (api) => {
+        api
+          .get(`/v3/stacks?&query={"org_uid":"${mockData.organizations[0].uid}"}`)
+          .reply(200, { stacks: mockData.stacks });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/environments').reply(200, { environments: mockData.environments });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/locales').reply(200, { locales: mockData.locales });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/stacks/branches').reply(200, { branches: mockData.branch });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/content_types?count=true').reply(200, { content_types: 2 });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/content_types?skip=0&include_branch=true').reply(200, { content_types: mockData.contentTypes });
+      })
+      .nock(cma, (api) => {
+        api
+          .get(
+            `/v3/content_types/${mockData.contentTypes[0].uid}/entries?include_publish_details=true&locale=${mockData.locales[0].code}&count=true`,
+          )
+          .reply(200, { entries: 1 });
+      })
+      .nock(cma, (api) => {
+        api
+          .get(
+            `/v3/content_types/${mockData.contentTypes[0].uid}/entries?include_publish_details=true&locale=${mockData.locales[0].code}&skip=0&limit=100&include_workflow=true`,
+          )
+          .reply(200, { entries: mockData.entry });
+      })
+      .command(['cm:export-to-csv'])
+      .it('Entries CSV file should be created with prompt');
   });
+});
 
-  it('Should show an error that user does not have org permissions to perform the operation if user enters such org', async () => {
-    const args = ['--action', 'users', '--org', process.env.ORG_WITH_NO_PERMISSION];
-    await ExportToCsvCommand.run(args);
-    assert.calledWith(errorStub, `You don't have the permission to do this operation.`);
+describe('export-to-csv with action users', () => {
+  describe('Export users csv file with flags', () => {
+    test
+      .stdout({ print: process.env.PRINT === 'true' || false })
+      .stub(fs, 'createWriteStream', () => new PassThrough())
+      .stub(mkdirp, 'sync', () => {})
+      .stub(process, 'chdir', () => {})
+      .nock(cma, (api) => {
+        api.get('/v3/user?include_orgs_roles=true').reply(200, { user: mockData.users[0] });
+      })
+      .nock(cma, (api) => {
+        api.get(`/v3/organizations/${mockData.organizations[0].uid}/roles`).reply(200, { roles: mockData.roles });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/user?include_orgs_roles=true').reply(200, { user: mockData.users[0] });
+      })
```

Comment:
Both user APIs look identical. Can we add `.persist()` and have only one nock method.?

Reply:
done